src/ddmd/dclass.d

```diff
@@ -1661,7 +1660,7 @@ extern (C++) final class InterfaceDeclaration : ClassDeclaration
             if (!baseclasses.dim && sc.linkage == LINKcpp)
                 cpp = true;
 
-            objc_InterfaceDeclaration_semantic_objcExtern(this, sc);
+            objc.setObjc(this, sc);
```

Comment:
Nit-pick, can we remove the scope parameter here?  It's only used for checking `sc.linkage`, which can be instead moved here.